index.html

```diff
@@ -1,311 +1,321 @@
+                    <ul class="social-icons">
+                        <li><img src="/img/social/Twitter.svg" alt=""></li>
+                        <li><img src="/img/social/Linkedin.svg" alt=""></li>
+                        <li><img src="/img/social/Vector-M.svg" alt=""></li>
+                        <li><img src="/img/social/Github.svg" alt=""></li>
+                        <li><img src="/img/social/Hand2.svg" alt=""></li>
+                    </ul>
+                </div>
+
+                <div class="btn-see-projects-about">
+                    <a href="#" class="button-about">Get my resume</a>
+                </div>
+            </div>
+
+            <div class="card-container-2">
+                <div class="languages-list-about">
+                    <ul>
+                        <li class="languages-list-names-about">
+                            <p>
+                                languages
+                                <img src="/img/Disabled.png" alt="row-icone" class="enabled-icon">
+                            </p>
+                            <ul class="flex-icone">
+                                <li class="container span-about">
+                                    <span class="icone-about">
+                                        <img src="/img/js.png" alt="js-icone" class="fav html">
+                                        <span class="label">JavaScript</span>
+                                    </span>
+                                </li>
+                                <li class="container span-about">
+                                    <span class="icone-about">
+                                        <img src="/img/html.png" alt="html-icone" class="fav html">
+                                        <span class="label">HTML</span>
+                                    </span>
+                                </li>
+                                <li class="container span-about">
+                                    <span class="icone-about">
+                                        <img src="/img/ic-css.png" alt="CSS-icone" class="fav css">
+                                        <span class="label">CSS</span>
+                                    </span>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="frameworks-about">
+                            <p class="frameworks">Frameworks</p>
+                            <img src="/img/Disabled-2.png" alt="row-icone" class="row-icone">
+                        </li>
+
+                        <li class="skills-about">
+                            <p class="skills">Skills</p>
+                            <img src="/img/Disabled-2.png" class="row-icone" alt="row-icone">
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+
+        <section class="contact-form">
```

Comment:
- [ ] I think it's better to fix the styling for the mobile version of the contact section. Your portfolio must be amazing for any size of the screen.
<img width="558" alt="Screen Shot 2022-08-27 at 11 13 51 PM" src="https://user-images.githubusercontent.com/47673311/187054609-c7464b5b-9833-4100-9b36-303ec30885f2.png">